src/docs/markdown/running.md

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
What if we call this file `run.md` instead? (`/docs/run`)

Reply:
Given the title is "Keep Caddy _Running_", feels better to have an actual word from the title as the URL? 🤔